data/src/main/java/pl/edu/agh/model/Availability.java

```diff
@@ -1,5 +1,40 @@
 package pl.edu.agh.model;
 
+import java.time.Duration;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Transient;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
 public class Availability {
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(nullable = false)
+    private Long id;
+
+    // TODO: hook up reference
+    @Transient
+    private UserInProject userInProject;
+
+    @ManyToOne
+    @JoinColumn(nullable = false)
+    private Sprint sprint;
+
+    private Duration timeAvailable;
+
+    private Duration delayFromPreviousSprint;
```

Comment:
Może `timeRemaining` po prostu?

Reply:
rozwazalem taka nazwe, ale uznalem ze blednie sugerowala by, ze dotyczy czegos zwiazanego z aktualnym sprintem

Reply:
Jest w tym racja. `leftOverTime`? `pastSprintRemainingTime`? [Mogę wymyślać więcej](https://i.redd.it/66ap5fx1iji01.png)

Reply:
`timeBacklog`?

Reply:
Backlog teoretycznie chyba oznacza to co trzeba, ale mi się w sumie scrumowo kojarzy inaczej

Reply:
to pastSprintRemainingTime jest chyba jeszcze najlepsze z tych wszystkich